Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -1757,6 +1758,24 @@ void addResult(HashMap<WebCore::FrameIdentifier, Ref<WebCore::LegacyWebArchive>>
     return { };
 }
 
+NSDictionary *WebPageProxy::getAccessibilityWebProcessDebugInfo()
+    {
+        const Seconds messageTimeout(2);
+        auto sendResult = protectedLegacyMainFrameProcess()->sendSync(Messages::WebPage::GetAccessibilityWebProcessDebugInfo(), webPageIDInMainFrameProcess(), messageTimeout);
+
+        if (!sendResult.succeeded())
+            return @{ };
+
+        auto [result] = sendResult.takeReplyOr(WebCore::AXDebugInfo({ 0, 0 }));
+
```

Comment:
Feels like there's an extra newline here. I'd be fine removing either the one above or the one below, but having both seems excessive.